sig/handwritten/app/models/user.rbs

```diff
@@ -3,4 +3,10 @@ class User < ApplicationRecord
   #       `#authentication_provider_github` is not optional association.
   def authentication_provider_github: () -> AuthenticationProviderGithub
   def have_unread_announcements?: () -> bool
+
+  class ActiveRecord_Relation
+    # NOTE: Added by kaminari
+    def page: (untyped) -> self
+    def per: (Integer) -> self
```

Comment:
Is `#per`, `#page` used anywhere in User?

Reply:
Steep tells me here.
https://github.com/kaigionrails/conference-app/blob/main/app/controllers/admin/users_controller.rb#L3

But I found there are no `#page` calls here. So we can choose to add only a type for `#per` to the `User` model. What do you think?

Reply:
Thanks for the explanation. I think we can add both `#per` and `#page`.
Because the methods are defined, whether it is used or not.